skimage/filters/_gabor.py

```diff
@@ -43,6 +43,8 @@ def gabor_kernel(frequency, theta=0, bandwidth=1, sigma_x=None, sigma_y=None,
         deviations
     offset : float, optional
         Phase offset of harmonic function in radians.
+    no_DC_offset: bool, optional, default False
```

Comment:
How should we call this internal function -- e.g. `_gabor_kernel` (even if it can also do Morlet wavelets)?